### ISSUE TITLE
Tutorial weave

### ISSUE THREAD

**Issue by miguelglezb** [pykoala/pykoala#415]
- In third cell, missing the file `../tests/weave/single_3042890.fit` 

- In fourth cell, no module named `pykoala.register`

**Comment by PabloCorcho** [comment]
I have not updated this tutorial following the latest changes of the pipeline. For the first release of the code this is not necessary as we will focus on KOALA. My suggestion it to leave it as it is for now (unless @paranoya wants to make use of it).

**Comment by PabloCorcho** [comment]
I have created a [branch](https://github.com/pykoala/koala/tree/414_fix_mplstyle) that includes the mplstyle in the MANIFEST.in file, so it is also included during the installation. Hopefully this solves this issue. @miguelglezb could you please try it in your environment?

**Comment by paranoya** [comment]
I have actually used some WEAVE data to test the sky-based calibrations in <https://github.com/pykoala/koala/pull/424>. However, it is still unclear whether those data are public, and therefore they'll sit on my disk for a while.
In any case, I'm not sure we should have separate tutorials for each instrument. Instead, I just set `example = 'KOALA'` or `example = 'WEAVE'` to specify the dataset to be read; the workflow is instrument-independent thereafter.
